source/extensions/filters/udp/dns_filter/dns_filter.cc

```diff
@@ -185,9 +185,8 @@ void DnsFilterEnvoyConfig::addEndpointToSuffix(const absl::string_view suffix,
   virtual_domains->emplace(std::string(domain_name), std::move(endpoint_config));
 
   auto success = dns_lookup_trie_.add(suffix, std::move(virtual_domains), false);
-  if (!success) {
-    throw EnvoyException(fmt::format("domain suffix [{}] already exists", suffix));
-  }
+  RELEASE_ASSERT(
```

Comment:
I think this can be a normal ASSERT

Reply:
Updated this to an `ASSERT`.   Fixed format and CI for docs completes successfully locally.